lib/record_store/changeset.rb

```diff
@@ -16,7 +16,6 @@ def self.removal(record)
       end
 
       def self.update(id, record)
-        raise ArgumentError.new('id cannot be nil') if id.nil?
         new(type: :update, record: record, id: id)
```

Comment:
What's the outcome if `id` is nil here?

Reply:
DynECT provider would error out on updates, but it shouldn't get to the point: (it wouldn't want to update a record that doesn't exist and any record that exists has an ID in DynECT).

Reply:
This was me being silly when originally building out the update logic.